ankihub/gui/operations/ankihub_sync.py

```diff
@@ -130,6 +131,24 @@ def _on_sync_done(future: Future, on_done: Callable[[Future], None]) -> None:
         send_review_data, on_done=_on_send_review_data_done
     )
 
+    last_summary_sent_date = config.get_last_summary_sent_date()
+    if not last_summary_sent_date:
+        last_summary_sent_date = (
+            datetime.now().replace(hour=0, minute=0, second=0, microsecond=0)
+            - timedelta(microseconds=1)
+        ) - timedelta(days=1)
+
+    feature_flags = config.get_feature_flags()
+    if (
+        feature_flags.get("daily_card_review_summary", False)
+        and last_summary_sent_date
```

Comment:
:pick: We can remove this line, because `last_summary_sent_date` is always set to a not-none value by the code above.
```suggestion
```